src/main/java/com/carrotcreative/cream/util/ObjectSizeUtil.java

```diff
@@ -0,0 +1,32 @@
+package com.carrotcreative.cream.util;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+
+public class ObjectSizeUtil
+{
+    private ObjectSizeUtil() {}
+
+    public static int getObjectSize(Serializable object)
```

Comment:
This could perhaps be in LruCacheEntry?  Maybe a helper function called `setSize()` that just goes and sets the size in the actual object itself.

Perhaps though this is the correct approach as we might use this in the future elsewhere -- just a thought!